data/migrations/20190110143613_recipes_table.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("recipes", table => {
+    table.increments();
+    table.string("name", 280).notNullable();
+    table
+        .integer("dishes_id")
+        .unsigned()
+        .references("id")
+        .inTable("dishes");
```

Comment:
There's a pretty good amount of repetition with the creation of these tables. What if we broke the majority of it out and used a helper function? For example, have this stored in a separate file:
```javascript
module.exports = (table, ref) => {
  table.increments();
  table.string('name', 280).notNullable();
  table
    .integer(`${ref}_id`)
    .unsigned()
    .references('id')
    .on(ref);
};
```
and then the usage in this migration would be:
```javascript
const makeMainTable = require('./path_to_helper')
exports.up = function(knex, Promise) {
    return knex.schema.createTable('recipes', table => {
        makeMainTable(table, 'dishes');
    })
};
```